android/android-email-sms.html

```diff
@@ -0,0 +1,332 @@
+  </style>
+  <script type="text/javascript">
+   //<![CDATA[
+   (function() {
+       var snippet = document.getElementById('firefox-android-sms-email-{{ snippet_id }}');
+       snippet.addEventListener('show_snippet', function() {
+           var input = document.querySelector('#firefox-android-sms-email-{{ snippet_id }} .number');
+           var button = document.querySelector('#firefox-android-sms-email-{{ snippet_id }} .submit-button');
+           var input_type;
+
+           // Checking input
+           input.addEventListener('input', function() {
+               input_type = checkInput(input.value);
+               if (input_type) {
+                   button.disabled = false;
+                   button.classList.remove('insensitive');
+               }
+               else {
+                   button.disabled = true;
+                   button.classList.add('insensitive');
+               }
+           });
+
+           // Submitting the form
+           var form = document.getElementById('snippet-{{ snippet_id }}-sms-email-send');
+           button.addEventListener('click', function(event) {
+               event.preventDefault();
+               if (input_type === 'email') {
+                   sendMetric('conversion-fennec-email', 
+                    function() { 
+                        sendEmailToBasket();
+                    });
+               } else if (input_type === 'phone') {
+                   sendMetric('conversion-fennec-sms', 
+                    function() { 
+                        sendSMSToBasket();
+                    });
+               } else {
+
+               }
+           });
+
+           // Logo change.
+           {% if replacement_logo %}
+             var newLogo = '{{ replacement_logo }}';
+             var brandLogo = document.getElementById('brandLogo');
+             if (brandLogo.src !== undefined) {
+                 brandLogo.src = newLogo;
+             } else {
+                 brandLogo.style.backgroundImage = 'url(' + newLogo + ')';
+             }
+             brandLogo.className += ' switch-{{ snippet_id }}';
+           {% endif %}
+
+           // Check if it is a valid email or phone number or neither
+           function checkInput (inp) {
+                var email_re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+                var check_email = email_re.test(inp);
+                // Light check for a valid US phone number
+                var check_phone = inp.length>=10 && inp.indexOf("@") === -1;
+
+                if ('{{ include_sms }}' !== 'True') {
+                    if (check_email) {
+                        return 'email';
+                    } else {
+                        return false;
+                    }
+                }
+
+                if (check_email) {
+                    return 'email';
+                } else if (check_phone) {
```

Comment:
`} else if (check_phone && '{{ include_sms }}' === 'True') ` can replace the lines 228-234